packages/atomic/cypress/integration/search-box/instant-results/search-box-instant-results.cypress.ts

```diff
@@ -0,0 +1,186 @@
+          },
+        })
+      )
+      .init();
+  }
+
+  describe('with keyboard navigation', () => {
+    before(() => {
+      setupWithSuggestionsAndRecentQueries();
+      SearchBoxSelectors.inputBox().type('{downarrow}{downarrow}{rightarrow}', {
+        delay: 500,
+        force: true,
+      });
+    });
+    SearchBoxAssertions.assertHasSuggestionsCount(maxRecentQueriesWithoutQuery);
+    InstantResultsAssertions.assertHasResultCount(numOfInstantResults);
+    CommonAssertions.assertAriaLiveMessage(
+      maxRecentQueriesWithoutQuery.toString()
+    );
+    describe('when navigating from query to results', () => {
+      after(() => {
+        SearchBoxSelectors.inputBox().clear({force: true});
+      });
+      InstantResultsAssertions.assertResultIsSelected(0);
+    });
+    describe('when navigating back from result to query', () => {
+      before(() => {
+        SearchBoxSelectors.inputBox().type(
+          '{downarrow}{downarrow}{rightarrow}{leftarrow}',
+          {
+            delay: 200,
+            force: true,
+          }
+        );
+      });
+      after(() => {
+        SearchBoxSelectors.inputBox().clear({force: true});
+      });
+      InstantResultsAssertions.assertNoResultIsSelected();
+      SearchBoxAssertions.assertSuggestionIsSelected(0);
+      SearchBoxAssertions.assertHasText('Recent query 0');
+    });
+
+    describe('when navigating back from result to query', () => {
+      before(() => {
+        SearchBoxSelectors.inputBox().type(
+          '{downarrow}{downarrow}{rightarrow}{leftarrow}',
+          {
+            delay: 200,
+            force: true,
+          }
+        );
+      });
+      after(() => {
+        SearchBoxSelectors.inputBox().clear({force: true});
+      });
+      InstantResultsAssertions.assertNoResultIsSelected();
+      SearchBoxAssertions.assertSuggestionIsSelected(0);
+      SearchBoxAssertions.assertHasText('Recent query 0');
+    });
+
+    describe('when pressing enter on a result', () => {
+      before(() => {
+        SearchBoxSelectors.inputBox().type(
+          '{downarrow}{downarrow}{rightarrow}{enter}',
+          {
+            delay: 200,
+            force: true,
+          }
+        );
+      });
+      it('redirects to new page', () => {
+        cy.window().then((win) => {
+          expect(win.location.href).to.equal('about:blank?0');
+        });
+      });
+    });
+  });
+
+  describe('with mouse navigation', () => {
+    before(() => {
+      setupWithSuggestionsAndRecentQueries();
+      SearchBoxSelectors.inputBox().click();
+    });
+
+    describe('when hovering over a query', () => {
+      before(() => {
+        SearchBoxSelectors.querySuggestions().eq(0).trigger('mouseover');
+      });
+      SearchBoxAssertions.assertSuggestionIsSelected(0);
+      SearchBoxAssertions.assertHasText('');
+
+      describe('when hovering over an instant result', () => {
+        before(() => {
+          cy.wait(1000);
```

Comment:
I would avoid `cy.wait` if possible.

Cypress already has [a mechanism](https://docs.cypress.io/guides/core-concepts/retry-ability) whenever we `find` or `get` to wait up to 4 seconds for the element to exist. If `InstantResultsSelectors.results().eq(1)` doesn't exist prior to this `wait`, Cypress should already await it.

Reply:
I think the problem here is that the element exists (but it's a different element, since we're finding it by index). I'll see what I can do!